thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -365,6 +367,8 @@ export class AccountManager {
         if (response.status === STATUS.OK) {
             const joinGameResponse: JoinGameResponse = await response.json();
             httpResponse.message = joinGameResponse.socketUrl;
+            // create the gamesocket here, after join game succeeds
+            GameSocket.createInstance(joinGameResponse.socketUrl);
```

Comment:
This can be removed since we changed to the component instantiating the socket, not AccountManager